setup.py

```diff
@@ -47,27 +47,28 @@ def run(self):
 
 
 setup(
-    name='WoeUSB-ng',
+    name='woeusb-ng',
     version='0.2.10',
-    description='WoeUSB-ng is a simple tool that enable you to create your own usb stick windows installer from an iso image or a real DVD. This is a rewrite of original WoeUSB. ',
+    description='woeusb-ng is a simple tool that enable you to create your own usb stick windows installer from an iso image or a real DVD. This is a rewrite of original WoeUSB.',
```

Comment:
The product name in the package description shouldn't be changed.

Reply:
ok